trymito.io/pages/infrastructure-integration-python-tool.tsx

```diff
@@ -0,0 +1,212 @@
+import type { NextPage } from 'next'
+import Image from 'next/image'
+import Head from 'next/head'
+import Footer from '../components/Footer/Footer';
+import Header from '../components/Header/Header';
+import homeStyles from '../styles/Home.module.css'
+import pageStyles from '../styles/Page.module.css'
+import titleStyles from '../styles/Title.module.css'
+import textImageSplitStyles from '../styles/TextImageSplit.module.css'
+import CTAButtons from '../components/CTAButtons/CTAButtons';
+import trifoldStyles from '../styles/Trifold.module.css'
+import { classNames } from '../utils/classNames';
+import FAQCard from '../components/FAQCard/FAQCard';
+import Link from 'next/link';
+import ContactCTACard from '../components/CTACards/ContactCTACard';
+
+const Home: NextPage = () => {
+
+  return (
+    <>
+      <Head>
+        <title>Python Infrastructure Integration Tools | Mito</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="description" content="Mito makes it easy to integrate cloud systems, data sources, custom functions, APIs and more. Connect IT infrastructure at enterprise scale with Mito." />
+      </Head>
+      
+      <Header/>
+    
+      <div className={pageStyles.container}>
+
+        <main className={pageStyles.main}>
+            <section className={classNames(pageStyles.background_card, titleStyles.title_card)}>
+                <h1 className={titleStyles.title}>
+                    Integrate Mito into your Enterprise Python Infrastructure
+                </h1>
+
+                <p className={titleStyles.description}>
+                    Let your analysts use custom Python functions and code snippets directly through the Mito spreadsheet
```

Comment:
```suggestion
                    Let analysts access your custom Python functions inside of a spreadsheet. 
```

I think _your_ being next to the analysts rather than custom Python functions is less clarifying.